assets/scss/_content.scss

```diff
@@ -39,6 +39,17 @@
       }
     }
   }
+  figure {
+    border: thin silver solid;
```

Comment:
Just to keep styling consistent... It looks better without the border. So, please, remove this line.
And also change `margin` and `padding` to `0`.